handbook/business-operations/README.md

```diff
@@ -296,10 +306,37 @@ Check the "📃 Planned articles" column in [#g-demand board](https://app.zenhub
 
 ## Rituals
 
+The following table lists this department's rituals, frequency, and Directly Responsible Individual (DRI).
```

Comment:
Please add this ritual to [business-operations.rituals.yml](https://github.com/fleetdm/fleet/blob/holidayn-patch-9/handbook/business-operations/business-operations.rituals.yml) and remove the table that was added back in this PR. (e.g. lines 310 - 332)